build.gradle.kts

```diff
@@ -112,29 +104,12 @@ tasks {
     jvmArgs("-Duser.language=en")
   }
 
-  withType<DokkaTask> {
-    configuration {
-      reportUndocumented = false
-      jdkVersion = 8
-      externalDocumentationLink {
-        url = URL("https://docs.spring.io/spring-framework/docs/5.1.9.RELEASE/javadoc-api/")
-        packageListUrl = URL(url, "package-list")
-      }
-      externalDocumentationLink {
-        url = URL("https://docs.spring.io/spring-boot/docs/2.1.x/api/")
-        packageListUrl = URL(url, "package-list")
-      }
-    }
-  }
-
```

Comment:
Hi, I don't understand why you deleted that. We don't want to lose the project documentation.

Reply:
Hi, I had inadvertently deleted it (during my tests).
Thanks for your return.